packages/common/test/templatization.test.ts

```diff
@@ -185,6 +185,33 @@ describe("Module `templatization`: common functions involving the adlib library"
       );
       expect(actual).toEqual(expected);
     });
+
+    it("should retain template variables when they are not in the templateDictionary", () => {
```

Comment:
With the adlib library admonition that [Transform functions should be VERY resilient - we recommend unit testing them extensively](https://github.com/Esri/adlib#notes-about-transforms), perhaps add test(s) to be sure that you get `{{${GEOCODE_SERVER_NAME}}}` when expected?

Reply:
@MikeTschudi the settings hash has no entry for any of these. The transform function no longer fails because of this and does return ` {{organization.helperServices.geocode:getDefaultLocatorURL}}` as now expected.